data/query_data.py

```
@@ -2,7 +2,8 @@
 from langchain_google_genai import GoogleGenerativeAI
 import argparse
 from langchain_chroma import Chroma
-from get_embedding_function import get_embedding_function
+from langchain.prompts import ChatPromptTemplate
```

Ruff (0.8.2)

`langchain.prompts.ChatPromptTemplate` imported but unused

Remove unused import: `langchain.prompts.ChatPromptTemplate`

(F401)